docs/documentation/server_admin/topics/user-federation/scim.adoc

```diff
@@ -0,0 +1,74 @@
+[[_scim]]
+
+=== SCIM client capabilities
+
+{project_name} includes a http://www.simplecloud.info[SCIM2] client allowing to :
+
+* Declare SCIM endpoints (through the identity federation UI). Any tool implementing SCIM protocol can be wired to the
+{project_name} instance through this declaration.
+* Propagate users and groups from {project_name} to SCIM endpoints : when a user/group gets created or modified in {project_name},
+the modification is forwarded to all declared SCIM endpoints through SCIM calls within the transaction scope. If
+propagation fails, changes can be rolled back or not according to a configurable rollback strategy.
+* Synchronize users and groups from SCIM endpoints (through the {project_name} synchronization mechanism).
+
+See https://datatracker.ietf.org/doc/html/rfc7643[RFC7643]
+and https://datatracker.ietf.org/doc/html/rfc7644[RFC7644] for further details
+
+==== Enabling SCIM extension
+
+[NOTE]
+====
+This extension is currently in experimental mode, and requires the ```SCIM``` experimental Profile to be enabled
+====
+
+.Procedure
+. Click on *Admin Console > Realm Settings > Events* in the menu.
+. Add `scim` to the list of event listeners
```

Comment:
Wouldn't be better if we skip this step and manage the event listener within the provider behind a setting? Or even automatically add the listener to the realm whenever a SCIM provider is created in the realm?

I'm trying to see the use cases where you don't want the listener enabled but still want to manually sync data by executing the different synchronization actions (e.g.: sync all users, sync changed users, etc).